S23/b-sai/main.py

```diff
@@ -0,0 +1,50 @@
+
+def to_lowercase(word: str, language: str):
+    """
+    word: str, the string to be converted to lowercase
+    language: str, the language of the string, in BCP 47 format
+    """
+    result = ""
+    
+    if language.startswith(("zh", "th", "ja")):
```

Comment:
There are 3-letter language code permitted in BCP-47, so "startswith" won't work here, e.g. "jam" is "Jamaican Creole English".